packages/gatsby/src/utils/babel-loader-helpers.ts

```diff
@@ -70,17 +94,17 @@ const prepareOptions = (babel, options = {}, resolve = require.resolve) => {
   )
 
   // Go through babel state and create config items for presets/plugins from.
-  const reduxPlugins = []
-  const reduxPresets = []
-  pluginBabelConfig.stages[stage].plugins.forEach(plugin => {
+  const reduxPlugins: PluginItem[] = []
+  const reduxPresets: PluginItem[] = []
+  pluginBabelConfig.stages[stage!].plugins.forEach(plugin => {
     reduxPlugins.push(
       babel.createConfigItem([resolve(plugin.name), plugin.options], {
         name: plugin.name,
         type: `plugin`,
       })
     )
   })
-  pluginBabelConfig.stages[stage].presets.forEach(preset => {
+  pluginBabelConfig.stages[stage!].presets.forEach(preset => {
```

Comment:
Same. If you've done a runtime check then this will have been narrowed to exclude null.